buildSrc/src/main/kotlin/io/getstream/chat/android/Dependencies.kt

```diff
@@ -33,14 +33,13 @@ object Versions {
     internal const val DOTENV = "5.2.2"
     internal const val ESPRESSO = "3.4.0"
     internal const val FRAGMENT_TESTING = "1.4.0"
-    internal const val FIREBASE_ANALYTICS = "19.0.0"
-    internal const val FIREBASE_CRASHLYTICS = "18.2.1"
-    internal const val FIREBASE_CRASHLYTICS_PLUGIN = "2.7.1"
-    internal const val FIREBASE_MESSAGING = "22.0.0"
+    internal const val FIREBASE_ANALYTICS = "21.0.0"
+    internal const val FIREBASE_CRASHLYTICS = "18.2.10"
```

Comment:
Should we mention these changes in the CHANGELOG? 🤔

Reply:
I thought it was only an internal dependency, but we provide it as a transitive dependency, so I will add a new change to the CHANGELOG